src/styles/eleventy-patterns/patterns/nav.scss

```diff
@@ -1,9 +1,145 @@
+.EP_header-container {
+  background: var(--EP_color-white);
+  box-shadow:
+    0 0 1px rgba(67, 90, 111, 0.47),
+    0 2px 4px -2px rgba(67, 90, 111, 0.3);
+  position: fixed;
+  top: 0;
+  transition: transform var(--transition-quick);
+  width: 100%;
+  z-index: 1;
+
+  @media (min-width: 800px) {
+    align-items: center;
+    display: flex;
+    height: var(--EP_nav-height);
+    justify-content: space-between;
+    padding: 0 1.5em;
```

Comment:
It would be nice to make more of these commons sizes (1.5, 1.15, etc.) into custom props. We could also use a modular scale. Maybe that's a follow-up issue?

Reply:
I think adding some common sizes is a good idea. I'm on the fence about using a modular scale. It could be nice to already have a sizing system set up. However, I'm unsure if people will want to use the same sizing that is being used in the project for the pattern library. What do you think?

Reply:
>However, I'm unsure if people will want to use the same sizing that is being used in the project for the pattern library.

I think we should do our best to keep our boilerplate CSS separate from the CSS our end users will be writing. The way we've generally handled this is with namespacing. If we create custom sizes, or implement a modular scale, we'd want to namespace that as well.

I think this is likely out of the scope of this PR though 🙂

Reply:
That makes sense, I can add an issue for this.

Reply:
Perfect, thanks